samples/javascript_nodejs/52.logger-bot/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "loggerBot",
+    "version": "1.0.0",
+    "description": "Demonstrate the transcript logging middleware in the nodejs SDK.",
+    "author": "Microsoft Bot Builder Yeoman Generator v4.0.10",
+    "license": "MIT",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build": "echo \"Error: no build specified\" && exit 1",
+        "start": "node ./index.js",
+        "watch": "nodemon ./index.js"
+    },
+    "dependencies": {
+        "botbuilder": "^4.0.6",
+        "botframework-config": "^4.0.6",
+        "cryptiles": "^4.1.2",
```

Comment:
Do we need this? Is this being used?

Reply:
Removed the cryptiles dependency, a few others were added due to switching to the multi-turn-prompt bot.